core-foundation-sys/src/error.rs

```diff
@@ -15,7 +15,7 @@ use string::CFStringRef;
 #[repr(C)]
 pub struct __CFError(c_void);
 
-pub type CFErrorRef = *mut __CFError;
+pub type CFErrorRef = *const __CFError;
```

Comment:
I'm not sure if this was `*mut` for a reason I have missed. But it was the only CF-ref typedef which was not `*const` so my guess is it was a typo. All tests of this crate and all dependant crates I tried work fine after this change.